manage_containers.py

```diff
@@ -74,12 +74,15 @@ def build(self):
 
     def rm(self):
         """Try to remove the container image if it exists"""
+        if not self.id:
+            return
+        print(f'\nRemove Ubuntu-{self.ubuntu} container image providing Clang {self.clang} and GCC {self.gcc}')
         try:
             cmd = self.runtime_cmd + ['rmi', '-f', self.id]
-            subprocess.run(cmd, text=True, check=True, stderr=subprocess.PIPE)
-            self.check()
+            subprocess.run(cmd, text=True, check=True)
         except:
-            print("[!] Warning, removal failed! Is image being used?")
+            print("[!] WARNING: image removing failed, see the error message above")
```

Comment:
By the way, what do you think about a slight visual improvement? I could paint the message in red color, then it will be better to catch the eye in the general pile of logs.